rust/connlib/clients/android/Cargo.toml

```diff
@@ -40,5 +40,8 @@ url = { workspace = true }
 [target.'cfg(target_os = "android")'.dependencies]
 android_log-sys = "0.3.1"
 
+[features]
```

Comment:
I don't like that we need to thread this all the way through. I'd rather require a build-time cfg via RUSTFLAGS env variable. That should be less code to maintain because it only needs to be dealt with in phoenix-channel.

Reply:
I just found out that `tokio-tungstenite` apparently supports a `rustls-tls-native-roots` feature flag directly, so we don't even need any code changes at all. And I switched to a `--cfg system_certs` directive.